nautobot/virtualization/filters.py

```diff
@@ -233,8 +296,29 @@ class VMInterfaceFilterSet(BaseFilterSet, StatusModelFilterSetMixin, CustomField
     mac_address = MultiValueMACAddressFilter(
         label="MAC address",
     )
-    tag = TagFilter()
+    tagged_vlans_vid = MultiValueCharFilter(method="filter_tagged_vlans_vid", label="Tagged VLAN (VID)")
```

Comment:
Could this instead be implemented without using `method` for the reasons outlined here: https://nautobot.readthedocs.io/en/stable/development/best-practices/#filtering-models-with-filtersets

Something like:
```py
tagged_vlans_vid = MultiValueCharFilter(field_name="tagged_vlans", lookup_expr="in", label="Tagged VLAN (VID)")
```